client/src/components/All_Modules_Page/index.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from "react";
```

Comment:
could you please change the name of the "index" file to something more intuitive,